### PR TITLE
UCT/RDMACM: enhance logging

### PR DESCRIPTION
**Pull request by alinask**

### CODE REVIEW

**Comment by alinask** [comment]
@yosefe take a look please.

File: src/uct/ib/rdmacm/rdmacm_cm.c [hunk 1]
```diff
@@ -113,13 +114,15 @@ static void uct_rdmacm_cm_handle_event_route_resolved(struct rdma_cm_event *even
         return;
     }
 
-    ucs_trace("%s: rdma_connect, cm_id %p",
-              uct_rdmacm_cm_ep_str(cep, ep_str, UCT_RDMACM_EP_STRING_LEN), cep->id);
+    ucs_trace("%s: rdma_connect, cm_id %p, on client %s",
```
Comment (yosefe) [inline src/uct/ib/rdmacm/rdmacm_cm.c]: on client %s -> from address %s

File: src/uct/ib/rdmacm/rdmacm_cm.c [hunk 2]
```diff
@@ -285,13 +288,19 @@ static void uct_rdmacm_cm_handle_event_connect_response(struct rdma_cm_event *ev
...
+    char                       remote_ip_port_str[UCS_SOCKADDR_STRING_LEN];
+    char                       client_ip_port_str[UCS_SOCKADDR_STRING_LEN];
     uct_device_addr_t          *dev_addr;
     size_t                     addr_length;
     uct_cm_remote_data_t       remote_data;
     ucs_status_t               status;
 
     ucs_assert(event->id == cep->id);
+    ucs_trace("client %s received connect_response from server %s",
+              ucs_sockaddr_str(rdma_get_local_addr(event->id), client_ip_port_str,
```
Comment (yosefe) [inline src/uct/ib/rdmacm/rdmacm_cm.c:302]: maybe use uct_rdmacm_cm_ep_str

File: src/uct/ib/rdmacm/rdmacm_cm_ep.c [hunk 3]
```diff
@@ -345,7 +346,9 @@ static ucs_status_t uct_rdamcm_cm_ep_server_init(uct_rdmacm_cm_ep_t *cep,
               event->id);
 
     if (rdma_accept(event->id, &conn_param)) {
-        ucs_error("rdma_accept(on id=%p) failed: %m", event->id);
+        ucs_error("rdma_accept(on id=%p) to client=%s failed: %m", event->id,
```
Comment (yosefe) [inline src/uct/ib/rdmacm/rdmacm_cm_ep.c]: uct_rdmacm_cm_ep_str?

**Comment by alinask** [comment]
@yosefe done.

File: src/uct/ib/rdmacm/rdmacm_cm.c [hunk 4]
```diff
@@ -113,13 +114,15 @@ static void uct_rdmacm_cm_handle_event_route_resolved(struct rdma_cm_event *even
         return;
     }
 
-    ucs_trace("%s: rdma_connect, cm_id %p",
-              uct_rdmacm_cm_ep_str(cep, ep_str, UCT_RDMACM_EP_STRING_LEN), cep->id);
+    ucs_trace("%s: rdma_connect, cm_id %p, from address %s",
+              uct_rdmacm_cm_ep_str(cep, ep_str, UCT_RDMACM_EP_STRING_LEN), cep->id,
```
Comment (yosefe) [inline src/uct/ib/rdmacm/rdmacm_cm.c:118]: shouldn't it be "to address" instead of "from address"?

File: src/uct/ib/rdmacm/rdmacm_cm_ep.c [hunk 5]
```diff
@@ -345,7 +346,10 @@ static ucs_status_t uct_rdamcm_cm_ep_server_init(uct_rdmacm_cm_ep_t *cep,
               event->id);
 
     if (rdma_accept(event->id, &conn_param)) {
-        ucs_error("rdma_accept(on id=%p) failed: %m", event->id);
+        ucs_error("%s: rdma_accept(on id=%p) to client=%s failed: %m",
```
Comment (yosefe) [inline src/uct/ib/rdmacm/rdmacm_cm_ep.c]: "%s: rdma_accept(id=%p client_address=%s) failed: %m",

File: src/uct/ib/rdmacm/rdmacm_cm.c (hunk 4 above)
Comment (alinask) [reply src/uct/ib/rdmacm/rdmacm_cm.c:118]: we are printing the address of the client that does 'rdma_connect' with rdma_get_**local**_addr so it's the address from which the connect request is sent to the server

File: src/uct/ib/rdmacm/rdmacm_cm_ep.c (hunk 5 above)
Comment (alinask) [reply src/uct/ib/rdmacm/rdmacm_cm_ep.c]: updated.

**Review by yosefe — APPROVED** [review]

**Pull request merged**